components/title/basic/src/index.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-basic: $c-white;
+$c-subtitle-basic: $c-white;
+
+$ff-roboto-sans-serif: 'Roboto', sans-serif; // Will be moved to nc-theme-basic
+
+$fw-title-basic: 500 !default;
+$fw-subtitle-basic: $fw-regular !default;
+
+$fz-title-basic: 48px !default;
+$fz-subtitle-basic: $fz-h2 !default;
+
+
+.sui-TitleBasic {
+  display: flex;
+  flex-direction: column;
+  font-family: $ff-roboto-sans-serif;
+
+  &-title {
+    color: $c-title-basic;
+    font-size: $fz-title-basic;
+    font-weight: $fw-title-basic;
+    margin: 0;
```

Comment:
Ok, I will remove the margins defined, just we need to remember to define the proper margins in the theme basic we create in the future for nc, or create "reset-margins" mixin in it.